o-fish-ios/Views/Components/PatrolBoatView/FindRecordsButtonView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FindRecordsButtonView.swift
+//  
+//  Created on 9/8/20.
+//  Copyright © 2020 WildAid. All rights reserved.
+//
+
+import SwiftUI
+
+struct FindRecordsButtonView: View {
+
+    var action: (() -> Void)?
+
+    private enum Dimensions {
+        static let spacing: CGFloat = 5.0
+        static let widthText: CGFloat = 89.0
+    }
+
+    var body: some View {
+        Button(action: action ?? {}) {
+            VStack(spacing: Dimensions.spacing) {
+                Image(systemName: "magnifyingglass")
+                    .font(.title2)
+                Text("Find Records")
+                    .font(.caption1)
+                    .frame(width: Dimensions.widthText)
```

Comment:
When the localised string is too large, this starts to look misplaced 
![image](https://user-images.githubusercontent.com/14197449/92727946-d7726180-f367-11ea-8bd7-623afcd0367e.png)

Reply:
Fixed